src/libcore/marker.rs

```diff
@@ -603,15 +603,99 @@ unsafe impl<T: ?Sized> Freeze for *mut T {}
 unsafe impl<'a, T: ?Sized> Freeze for &'a T {}
 unsafe impl<'a, T: ?Sized> Freeze for &'a mut T {}
 
-/// Types which can be moved out of a `PinMut`.
+/// A trait that indicates that it is safe to move an object of a type implementing it.
```

Comment:
Rustdoc docs really like

```rust
/// A single sentence summary
///
/// Real docs start here...
```

The old code was in this style, this patch throws it out.